registry/users.go

```diff
@@ -121,18 +120,3 @@ func validateSelf(s *envelope.User) error {
 
 	return nil
 }
-
-type omit struct{}
```

Comment:
This should have been `type omit *struct{}` :(

I like what you've done muuuch better though